tracer/src/Datadog.Trace/IAST/SourceType.cs

```diff
@@ -0,0 +1,29 @@
+// <copyright file="SourceType.cs" company="Datadog">
+// Unless explicitly stated otherwise all files in this repository are licensed under the Apache 2 License.
+// This product includes software developed at Datadog (https://www.datadoghq.com/). Copyright 2017 Datadog, Inc.
+// </copyright>
+
+namespace Datadog.Trace.Iast;
```

Comment:
turn on: `#nullable enable` in new files please.

Reply:
Done!